specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/preview/2018-09-01/BackupLocations.json

```diff
@@ -0,0 +1,325 @@
+                    },
+                    {
+                        "$ref": "#/parameters/BackupLocationParameter"
+                    },
+                    {
+                        "$ref": "Backup.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "Backups.json#/definitions/Backup"
+                        }
+                    },
+                    "202": {
+                        "description": "ACCEPTED",
+                        "schema": {
+                            "$ref": "Backups.json#/definitions/Backup"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "BackupLocation": {
+            "description": "Information about the backup location.",
+            "type": "object",
+            "properties": {
+                "properties": {
+                    "description": "Properties of a backup location.",
+                    "$ref": "#/definitions/BackupLocationModel",
+                    "x-ms-client-flatten": true
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "Backup.json#/definitions/Resource"
+                }
+            ]
+        },
+        "BackupLocationModel": {
+            "type": "object",
+            "description": "Properties of a backup location.",
+            "properties": {
+                "externalStoreDefault": {
+                    "description": "Information about an external storage location.",
+                    "$ref": "#/definitions/ExternalStore",
+                    "x-ms-client-flatten": true
+                }
+            }
+        },
+        "ExternalStore": {
+            "type": "object",
+            "description": "Information about an external storage location.",
+            "properties": {
+                "path": {
+                    "description": "Path to the update location",
+                    "type": "string"
+                },
+                "userName": {
+                    "description": "Username to access the location.",
+                    "type": "string"
+                },
+                "password": {
+                    "description": "Password to access the location.",
+                    "type": "string"
+                },
+                "encryptionCertBase64": {
+                    "description": "The base64 raw data for the backup encryption certificate.",
+                    "type": "string"
+                },
+                "encryptionCertThumbprint": {
+                    "description": "The thumbprint of the encryption certificate.",
+                    "type": "string",
+                    "readOnly": true
+                },
```

Comment:
Added encryptionCertBase64 and encryptionCertThumbprint, and removed encryptionKeyBase64. Others are the same as 2016-05-01.

Reply:
Same question as above.